Enterprise-CMCS/macpro-mdct-mcr
# add error handling

## Description

Adds some basic error handling so we can catch those errors and show 'em to the people.

### How to test

1. Pull and run.
2. Try to log in with invalid credentials. You should get a browser alert with an error message.

### Changed Dependencies

No changed deps.

## Code author checklist

## Reviewer checklist (two different people)
